File: src/scitrader.marketdataserver/src/main/java/com/scitrader/marketdataserver/common/ArgumentException.java
package com.scitrader.marketdataserver.common;

public class ArgumentException extends RuntimeException {
        public ArgumentException() {
                super();
        }

        public ArgumentException(String message) {
                super(message);
        }

        public ArgumentException(String message, Exception inner){
                super(message, inner);
        }

        public ArgumentException(Exception inner){
                super(inner);
        }
}

File: Kotlin/hello-kotlin-world/src/main/java/com/scitrader/marketdataserver/common/ArgumentException.kt
package com.scitrader.marketdataserver.common

open class ArgumentException : RuntimeException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, inner: Exception) : super(message, inner) {}

    constructor(inner: Exception) : super(inner) {}
}

